content/operate/rc/resilient-apps.md

```diff
@@ -0,0 +1,90 @@
+
+To create Active-Active databases, you need to create Redis Cloud Pro subscription and enable Active-Active Redis and define the regions for each copy of your databases. See [Create an Active-Active database]({{< relref "/operate/rc/databases/create-database/create-active-active-database" >}}) for instructions.
+
+### Set manual maintenance windows
+
+Redis maintains your Redis Cloud subscriptions and databases as needed to ensure your databases are running the most stable and up-to-date version of Redis. By default, Redis will perform [maintenance]({{< relref "/operate/rc/subscriptions/maintenance" >}}) automatically while limiting service disruption as much as possible.
+
+For stable apps, you may want to control when Redis can perform maintenance on your databases. For Redis Cloud Pro subscriptions, you can [set manual maintenance windows]({{< relref "/operate/rc/subscriptions/maintenance/set-maintenance-windows" >}}) to ensure non-urgent maintenance will occur at set times. Configuring or altering the maintenance window will not have any impact on your subscription or databases.
+
+### Enable TLS
+
+Transport Layer Security (TLS) uses encryption to secure network communications. 
+
+Because TLS has an impact on performance, you need to determine whether the security benefits of TLS are worth the performance impact. TLS recommendations depend on the subscription plan and whether clients connect to your database using public or private endpoints.
+
+This table shows TLS recommendations:
+
+| Subscription | Public&nbsp;endpoint | Private endpoint |
+|--------------|----------------------|------------|
+| Redis Cloud Essentials        | Enable TLS           | N/A |
+| Redis Cloud Pro     | Enable TLS           | Enable TLS if security outweighs performance impact |
+
+## Set up Redis clients
+
+When you're developing your apps, we recommend that you use specific Redis Client features to connect to Redis Cloud if they are available for your preferred client.
+
+See [Clients]({{< relref "/develop/connect/clients/" >}}) to learn how to connect with the official Redis clients.
+
+### Re-attempt connections
+
+Some clients allow you to re-try connecting to your database if the connection fails. For these clients, we recommend that you implement connection re-attempts to ensure high availability and connection stability. 
+
+For example, `redis-py` uses the [Retry](https://redis-py.readthedocs.io/en/stable/retry.html) class to implement connection retries.
+
+### Refresh DNS
+
+Your application may disconnect from your database either during planned maintenance or for other, unplanned reasons. Most Redis clients are set to refresh their DNS address when they reconnect to the database, and you will not be required to perform any further action. If you encounter connectivity problems for more than a minute during maintenance, please refresh your DNS entries. 
```

Comment:
I think we tend to avoid "please", going by the [Google guide](https://developers.google.com/style/tone#politeness-and-use-of-please).
```suggestion
Your application may disconnect from your database either during planned maintenance or for other, unplanned reasons. Most Redis clients are set to refresh their DNS address when they reconnect to the database, and you will not be required to perform any further action. If you encounter connectivity problems for more than a minute during maintenance then you should refresh your DNS entries. 
```